jenkins/sign-artifacts/sign-standalone-artifacts.jenkinsfile.txt

```diff
@@ -0,0 +1,33 @@
+   sign-standalone-artifacts.run()
```

Comment:
Please move this file into the `.../tests/data` path with the other files

Reply:
data was being used to store dummy files that are required for tests. I have created a separate folder to hold regression txt files for jenkins job, `tests/jenkins/jenkinsjob-regression-files`. Let me know if the name of the dir works